chat-plugins/polls.js

```diff
@@ -0,0 +1,102 @@
+/* Poll commands plugin
+ * This is a poll commands system that will allow a voice (or roomvoice) to host 
+ * polls in terms of starting, reminding
+ * by: panpawn
+ */
+
+function splint(target) {
+	var cmdArr = target.split(",");
+	for (var i = 0; i < cmdArr.length; i++) cmdArr[i] = cmdArr[i].trim();
+		return cmdArr;
+}
+
+exports.commands = {
+
+	survey: 'poll',
+	poll: function (target, room, user) {
+		if (!this.canBroadcast()) return;
+		if ((user.locked || user.mutedRooms[room.id]) && !user.can('bypassall')) return this.sendReply("You cannot do this while unable to talk.");
+		if (room.question) return this.sendReply('There is currently a poll going on already.');
+		if (!target) return false;
+		if (target.length > 500) return this.sendReply('Polls can not be this long.');
+		var separacion = "&nbsp;&nbsp;";
+		var answers = splint(target);
+		if (answers.length < 3) return this.sendReply('Correct syntax for this command is /poll question, option, option...');
+		var question = answers[0];
+		question = Tools.escapeHTML(question);
+		answers.splice(0, 1);
+		var answers = answers.join(',').toLowerCase().split(',');
+		room.question = question;
+		room.answerList = answers;
+		room.usergroup = Config.groupsranking.indexOf(user.group);
+		var output = '';
+		for (var u in room.answerList) {
+			if (!room.answerList[u] || room.answerList[u].length < 1) continue;
+			output += '<button name="send" value="/vote ' + room.answerList[u] + '">' + Tools.escapeHTML(room.answerList[u]) + '</button>&nbsp;';
+		}
+		room.addRaw('<div class="infobox"><h2>' + Tools.escapeHTML(room.question) + separacion + '<font size=2 color = "#939393"><small>/vote OPTION<br /><i><font size=1>Poll started by ' + Tools.escapeHTML(user.name) + '</font size></i></small></font></h2><hr />' + separacion + separacion + output + '</div>');
```

Comment:
For most of the uses I've seen where `<br />` have been used on showdown, it's the wrong use for it.
Instead, use block elements instead. e.g., `<p>`, `<h1>`, `<div>`